endpoints.json

```diff
@@ -42,7 +42,26 @@
           }
         ]
       }
+    },
+    "/api/articles/:article_id/comments (POST)": {
```

Comment:
This should follow the same convention as the original given endpoints, so it should start with `"<Method> <endpoint>"`